libmate-panel-applet/panel-plug-private.h

```diff
@@ -18,6 +18,12 @@
 #ifndef PANEL_PLUG_PRIVATE_H
 #define PANEL_PLUG_PRIVATE_H
 
+#include <config.h>
```

Comment:
Is this include here really needed?

Reply:
It pulls in the `HAS_X11` define, which is checked to insure this file is not being built under incorrect circumstances.

Reply:
Well I have not seen that someone would include `config.h` in header files... Also when `panel-plug.c` is compiled `config.h` is included first so by the time this file is included that define should be already available, no?

Should not it be more or less same as it would have been done this way:
```
#define HAS_X11
#include "panel-plug-private.h"
```